website/sidebars.json

```diff
@@ -23,6 +23,7 @@
       "searchbar",
       "slider",
       "social_icon",
+      "textSegment",
```

Comment:
This fails right now. Instead of using `textSegment` for the id, use `text_segment`

Reply:
i will use text_segment replace of textSegment